lib/srv/alpnproxy/kube.go

```diff
@@ -220,23 +228,38 @@ func (m *KubeMiddleware) OverwriteClientCerts(req *http.Request) ([]tls.Certific
 var ErrUserInputRequired = errors.New("user input required")
 
 // reissueCertIfExpired checks if provided certificate has expired and reissues it if needed and replaces in the middleware certs.
+// serverName has a form of <hex-encoded-kube-cluster>.<teleport-cluster>.
```

Comment:
I believe the actual implementation works this way, however I had to update the tests because they only passed a kube cluster name here. @tigrato were the tests wrong, or is this assumption wrong?

Reply:
Cool, so the test should be correct now.